worlds/rogue_legacy/Regions.py

```diff
@@ -100,24 +95,18 @@ def create_regions(multiworld: MultiWorld, player: int):
     multiworld.get_entrance("The Fountain Room", player).connect(multiworld.get_region("The Fountain Room", player))
 
 
-def create_region(multiworld: MultiWorld, player: int, name: str, locations=None, exits=None):
-    ret = Region(name, RegionType.Generic, name, player)
-    ret.multiworld = multiworld
+def create_region(multiworld: MultiWorld, player: int, name: str, locations=None, region_exits=None):
```

Comment:
This could be entirely reduced to `World` and `region_name` as args.

Reply:
Still need region data as regions are generated when `create_regions` is called and not ahead of time.